CHANGELOG.md

```diff
@@ -5,19 +5,22 @@ NOTE: We are deprecating several collectors in this release.
     * `megacli` - Requires forking, moved to textfile collection.
```

Comment:
@SuperQ  This is in past tense, but the collector hasn't been moved yet, or am I missing something? What about `will be moved to ...` instead?